lib/google-contacts/element.rb

```diff
@@ -289,7 +309,10 @@ def update_groups(*group_links)
     end
 
     private
-     # Evil ahead
+      def get_google_type(google_type)
```

Comment:
@sampatbadhe should it be `get_google_label_name` or anything else that is more relevant? Method name `get_google_type` is very confusing and its not clear what it returns or does. It just has comment `# Evil ahead`